services/121-service/src/financial-service-providers/financial-service-provider-configuration.mapping.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  FinancialServiceProviderConfigurationEnum,
+  FinancialServiceProviderName,
+} from '@121-service/src/financial-service-providers/enum/financial-service-provider-name.enum';
+
+// ##TODO: Merge this mapping with the one in financial-service-providers.const.ts??? To be decide when implementing the FSP config api
+export const FinancialServiceProviderConfigurationMapping: {
```

Comment:
Yah, I think it can be part of that const we defined. I put that in the TODO comment for the task 'Create/refactor APIs according to miro'. So putting this comment to resolve for now. Let me know if you disagree @diderikvw